docs/installing-northstar/northstar-installers/vtol-guide.md

```diff
@@ -0,0 +1,120 @@
+# VTOL Guide
+
+VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2. You can find links to install it on the [Northstar Website](https://northstar.tf), or on [VTOL's GitHub page](https://github.com/R2NorthstarTools/VTOL).
+
+## Main Menu
+
+![vtol-main-window](https://user-images.githubusercontent.com/70904206/219880640-b50e8ff4-4db1-4bb9-8bd2-929582783083.png)
+**The Main Menu of VTOL is where you'll probably spend most of your time.**
+
+This menu has a lot of useful information, such as the version of VTOL you have, the version of Northstar you have, if the Master Server is online, whether EA App/Origin are currently open, and a place to set your game path. 
+
+## Version Numbers
+
+You can see what version of VTOL/Northstar you have in the top left corner of the VTOL window.
+![version-numbers](https://user-images.githubusercontent.com/70904206/219880616-eed5a09a-0b6f-482b-9274-f64b4191a6e4.png)
```

Comment:
The images should be committed as to not depend on external resources. You can put them in the `docs/images/` directory. Check the source of other pages for how to link to them ^^